server.js

```diff
@@ -1,6 +1,6 @@
 const path = require('path');
 const express = require('express');
-const config = require('./.config');
```

Comment:
Andrew , 

Please leave this as ./.config.js    and rename your config file to .config.js. As Ash mentioned it is a convention to add a  .  before hidden file names.  
Also, in git ignore make sure the ignored file is .config.js

Reply:
Thank you for good point guys.
I did rename it in my branch to ./.config